src/screens/home/components/HomeView.js

```diff
@@ -114,6 +125,35 @@ const HomeView = ({ route, navigation, setLauncherContext, setBarStyle }) => {
     }
   }, [uri, client, route, navigation, session])
 
+  useEffect(
```

Comment:
I think we should strive to not add any more logic into view components at this point.
Using the pure functions (when possible) with hook-as-a-bridge seem like a good replacement.
Also anyway the logic that I see here should not be triggered by a View in any case (seems higher level in terms of abstraction), but I might have missed something

Reply:
> the logic that I see here should not be triggered by a View in any case (seems higher level in terms of abstraction), but I might have missed something

Here I got the same problem you had in the past for the lockscreen.
If the initial route is `drive` we want to display the CozyAppScreen, but we still want to load the HomeScreen in background. We also want the back button to correctly go back to the `home` route.

The solution I found was to navigate to `home` first then the home detects it should open `drive`.

However I agree if you want to split this component into a view/hook organisation. I propose to do it in another PR

Reply:
Yeah agree for the render problem.

Ok for refactoring later